src/main/java/tasklist/TaskList.java

```diff
@@ -0,0 +1,324 @@
+     *
+     * @param items represents the list of task
+     * @param scanner the scanner object to read next line of input
+     * @return returns the next input command by user
+     */
+    public static String checkList(List<Task> items, Scanner scanner) {
+        int index = 1;
+        StringBuilder sbr = new StringBuilder("Here are the tasks in your list: \n");
+        System.out.println("Here are the tasks in your list:");
+        for (Task item : items) {
+            System.out.println(index + "." + item);
+            sbr.append(index + "." + item + "\n");
+            index++;
+        }
+        //return scanner.nextLine();
+        return sbr.toString();
+    }
+
+    /**
+     * Marks a particular task with that specific index as done
+     *
+     * @param input represents the index of the task to mark
+     * @param items represents the list of task
+     * @param scanner the scanner object to read next line of input
+     * @return returns the next input command by user
+     */
+    public static String markingTask(String input, List<Task> items, Scanner scanner) {
+        try {
+            int taskIndex = Integer.parseInt(input) - 1;
+            Task markingTask = items.get(taskIndex);
+            markingTask.setDone(true);
+            System.out.println("____________________________________________________________");
+            System.out.println("Nice! I've marked this task as done:");
+            System.out.println("[" + markingTask.getStatusIcon() + "] " + markingTask.getDes());
+            System.out.println("____________________________________________________________");
+            //return scanner.nextLine();
+            return "Nice! I've marked this task as done:\n" +
+                    "[" + markingTask.getStatusIcon() + "] " + markingTask.getDes() + "\n";
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println("Might want to reconsider your action. Please Try Again");
+            //return scanner.nextLine();
+            return "Might want to reconsider your action. Please Try Again";
+        } catch (NumberFormatException e) {
+            System.out.println("Might want to reconsider your action. Please Try Again");
+            //return scanner.nextLine();
+            return "Might want to reconsider your action. Please Try Again";
+        }
+    }
+
+    /**
+     * Unmarks a particular task with that specific index as done
+     *
+     * @param input represents the index of the task to unmark
+     * @param items represents the list of task
+     * @param scanner the scanner object to read next line of input
+     * @return returns the next input command by user
+     */
+    public static String unmarkingTask(String input, List<Task> items, Scanner scanner) {
+        try {
+            int taskIndex = Integer.parseInt(input) - 1;
+            Task markingTask = items.get(taskIndex);
+            markingTask.setDone(false);
```

Comment:
Any reason why this is named `markingTask` instead of something like `unmarkingTask`?